docs/getting_started/dask.rst

```diff
@@ -1,91 +1,104 @@
-Modin vs. Dask Dataframe
+Modin vs. Dask DataFrame
```

Comment:
Should we remove docs/modin_vs_dask.rst and rename this file from dask.rst to modin_vs_dask.rst?

Reply:
I will do this together in #3792 when I do the restructuring of the sections.

Reply:
removed `docs/modin_vs_dask.md` and renamed `dask.rst` to `modin_vs_dask.rst`